index.html

```diff
@@ -4,42 +4,175 @@
     </nav>
 </header>
 
 
-<section>
+<section class="item-1">
     <div class="headline">
 
         <h1 class="intro name">Hey There. I'm Chidiebere Ojingwa</h1>
-        <p class="intro profession">I'am a Software Developer</p>
+        <h2 class="intro profession">I'am a Software Developer</h2>
         <p class="intro task"> I can help you build a product, feature or <br> website
-        look through some of my work <br>and experience! If you like what you see <br> and
-        have a project you need, <br> don't hesitate to contact me</p>
+            look through some of my work <br>and experience! If you like what you see <br> and
+            have a project you need, <br> don't hesitate to contact me</p>
 
         <ul class="social">
             <li><a href="#" class="fa fa-github"></a></li>
             <li><a href="#" class="fa fa-linkedin"></a></li>
             <li><a href="#" class="fa fa-angellist"></a></li>
             <li><a href="#" class="fa fa-twitter"></a></li>
-            <li> <a href="#" class="fa fa-medium"></a></li>
+            <li><a href="#" class="fa fa-medium"></a></li>
+        </ul>
+    </div>
+</section>
+
+<section class="item-2">
+    <!-- <div class="recentWork"> -->
+        <h2 class="recentWork">My Recent Works</h2>
+    <!-- </div> -->
+    <img class="line" src="assets/line.png" alt="straight line">
+    <div class="works first">
+        <img class="firstProjectImg" src="./assets/imgplaceholder.png" alt="project Image">
+        <h2 class="multi"> Multi-Post Stories</h2>
+        <p class="story">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="grid-container">
+            <li class="stack">css</li>
+            <li class="stack">html</li>
+            <li class="stack">bootstrap</li>
+            <li class="stack">Ruby</li>
         </ul>
+        <a class="project">See Project</a>
+    </div>
+
+
+    <div class="second pos">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="grid-container2">
+            <li class="stack2">html</li>
+            <li class="stack2">bootstrap</li>
+            <li class="stack2">Ruby</li>
+        </ul>
+        <a href="#" class="button">See Project</a>
+    </div>
+
+    <div class="third pos">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
```

Comment:
- [ ] As requested by the former code reviewer, which is also part to the project requirements, the project technologies should be a list `<ul>` and each of them is a list item `<li>`

Reply:
already implemented